sdk/src/main/java/io/dapr/client/DaprClientHttp.java

```diff
@@ -662,6 +665,110 @@ public Mono<Map<String, Map<String, String>>> getBulkSecret(GetBulkSecretRequest
         .map(m -> (Map<String, Map<String, String>>) m);
   }
 
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public Mono<Boolean> tryLock(LockRequest request) {
+    try {
+      final String stateStoreName = request.getStoreName();
+      final String resourceId = request.getResourceId();
+      final String lockOwner = request.getLockOwner();
+      final Integer expiryInSeconds = request.getExpiryInSeconds();
+
+      if ((stateStoreName == null) || (stateStoreName.trim().isEmpty())) {
+        throw new IllegalArgumentException("State store name cannot be null or empty.");
+      }
+      if (resourceId == null || resourceId.isEmpty()) {
+        throw new IllegalArgumentException("ResourceId cannot be null or empty.");
+      }
+      if (lockOwner == null || lockOwner.isEmpty()) {
+        throw new IllegalArgumentException("LockOwner cannot be null or empty.");
+      }
+      if (expiryInSeconds < 0) {
+        throw new IllegalArgumentException("ExpiryInSeconds cannot be negative.");
+      }
+
+
+      Map<String, Object> jsonMap = new HashMap<>();
+      jsonMap.put("resourceId", resourceId);
+      jsonMap.put("lockOwner", lockOwner);
+      jsonMap.put("expiryInSeconds", expiryInSeconds);
```

Comment:
Would you mind making these keys constants? Magic strings / numbers can cause bugs due to typos. I think @mukundansundar's suggestion below would also help with this.